artifacts/deploy/webhook-configuration.yaml

```
@@ -237,3 +237,20 @@ webhooks:
     sideEffects: None
     admissionReviewVersions: [ "v1" ]
     timeoutSeconds: 3
+  - name: resourcedeletionprotection.karmada.io
```

```suggestion
  - name: resource deletion protection
```
Just to eliminate misleading as this validating hook is not for a concrete API object.